tsconfig.json

```diff
@@ -8,8 +8,7 @@
     "strict": true,
     "rootDir": "src",
     "esModuleInterop": true,
-    "noEmit": true
   },
-  "include": ["src", "vscode.d.ts"],
+  "include": ["src"],
   "exclude": ["node_modules"]
```

Comment:
would be cool if we could keep the vscode.d.ts for now inside this PR like we had before. We are happy to get rid of it I believe, but if so, then in a separate PR. Thanks!

Reply:
Another PR sounds more reasonable.